remoulade/__init__.py

```diff
@@ -68,4 +69,4 @@
     "Scheduler", "ScheduledJob", "get_scheduler", "set_scheduler"
 ]
 
-__version__ = "0.15.3"
```

Comment:
don't care about the version it's done in aontehr commit using https://pypi.org/project/bumpversion/. I'll do it when deploying

Reply:
but you can fill changelog.rst ;)